src/fsharp/TypeChecker.fs

```diff
@@ -4555,9 +4555,9 @@ and TcTyparOrMeasurePar optKind cenv (env: TcEnv) newOk tpenv (Typar(id, _, _) a
                         elements
                         |> Seq.map (fun p -> "'" + p.Key)
 
-                [ yield! predictions1 
-                  yield! predictions2 ]
-                |> HashSet
+                seq {
```

Comment:
If we can by pass these seq concatenations and instead fold the seq maps above into a buffer it will be way more efficient?

Reply:
the seq is already gone. refresh the diff view